### PR TITLE
fixed the num_samples of text classification model.

### PR DESCRIPTION
**Pull request by VahidooX**

### CODE REVIEW

**Review by tkornuta-nvidia — DISMISSED** [review]
LGTM

File: examples/tts/tacotron2.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
```
[the remark this answers is not in the document]
Comment (VahidooX) [reply examples/tts/tacotron2.py:2]: It was added in 2020 but they forgot to add the headers. Should I use 2021?

File: nemo/collections/nlp/data/text_classification/text_classification_dataset.py [hunk 2]
```diff
@@ -116,8 +116,13 @@ def __init__(
...
                 else:
                     with open(input_file, "r") as f:
-                        lines = f.readlines(num_samples)
+                        lines = f.readlines()
                         logging.info(f'Read {len(lines)} examples from {input_file}.')
+                        if num_samples > 0:
+                            lines = lines[:num_samples]
+                            logging.warning(
+                                f"Parameter 'num_samples' is set, so just the first {len(lines)} examples are kept."
+                            )
```
[the remark this answers is not in the document]
Comment (VahidooX) [reply nemo/collections/nlp/data/text_classification/text_classification_dataset.py:125]: @tkornuta-nvidia has asked for it. I think they use it for TLT.

**Review by ekmb — APPROVED** [review]

File: examples/tts/tacotron2.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (VahidooX) [reply examples/tts/tacotron2.py:2]: fixed

**Review by tkornuta-nvidia — APPROVED** [review]

**Pull request merged**